lib/io/file/move.fz

```diff
@@ -39,6 +39,17 @@ private move(
         replace
         tmp
 
+# short-hand for creating and installing effects
+#
+move_custom(f () -> unit) => move_custom default_move_operation f
```

Comment:
is it not possible to also call this `move`?

Reply:
it is, but i wanted to keep some consistency in naming.